### DISCUSSION TITLE
关于当前网络中文拼音首字母缩写风潮的一些探讨

### DISCUSSION THREAD

**Discussion by j1g5awi** [sparanoid/chinese-copywriting-guidelines#139] [Ideas]
当今网络上，中文拼音首字母缩写的使用已经可以说是家常便饭了。

一开始可能只是防和谐需要，除了一些涉政的名字，还有国骂”他妈的“经常被缩写成 TMD，”傻逼“被缩写成 SB，”你妈逼“被缩写成 NMB。但目前来看，中文拼音首字母缩写的风潮已经进入下一阶段，出现了各种无法理解的现象，就像当年的火星文风潮，既无用处，也无逻辑。

我一直觉得中文本身就是一门简练的语言，如果有必要进行进一步的缩写，那也应该是如“计算机科学与技术”缩写成“计科”这样的缩写，而非拉丁文式的首字母缩写。目前比较常见的缩写如 yygq（阴阳怪气）、xswl（笑死我了）、bksn（不愧是你）等，第一次接触的人无法快速理解（觉得前面这些容易理解的可以尝试理解下 bbl 是什么意思），对于人与人之间交流毫无帮助。为了解释这些日益增多的拼音首字母缩写，甚至还有朋友做了个词典 [nbnhhsh](https://github.com/itorr/nbnhhsh)，说严重点纯属语言系统上的倒退。

简而言之，中文拼音首字母缩写在某种程度上让中文使用者之间的交流变得困难。

鉴于此，**本排版指北是否有必要应该新增一条关于此的原则，强调在排版中尽量少用中文拼音首字母缩写**？

欢迎各位留言探讨。

**Comment by liyishuai** [comment]
原则上，拼音不属于中文，只是现代标准汉语的读音表记法，而非替代文字，上过小学的人应禁止使用。

例外：汉语水平考试（HSK）等涉外概念，可声明后使用其标准缩写。
